vite.config.ts

```diff
@@ -19,6 +19,18 @@ export default defineConfig({
       jsxRuntime: 'classic'
     })
   ],
+  'build': {
+    'chunkSizeWarningLimit': 4096,
+    'rollupOptions': {
+      'output': {
+        'manualChunks': {
+          'morpheus-app': ['morpheus-app'],
+          'plotly.js-dist': ['plotly.js-dist'],
+          'igv': ['@single-cell-portal/igv']
```

Comment:
At a glance, I wondered "What's this?".  Your rather excellent exposition in this PR's description is among the best answers.  I'd add a tiny explanation inline, and link to the fuller one.

```suggestion
        // Safely split out especially large third-party libraries
        // Fuller explanation: https://github.com/broadinstitute/single_cell_portal_core/pull/1668
        'manualChunks': {
          'morpheus-app': ['morpheus-app'],
          'plotly.js-dist': ['plotly.js-dist'],
          'igv': ['@single-cell-portal/igv']
```